<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surface Reconstruction</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="imagesPreviewScrollArea">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="imagesPreviewScrollAreaContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>198</width>
           <height>701</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="imagesPreviewArea">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="activeProjectInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>110</height>
           </size>
          </property>
          <property name="title">
           <string>Active project info</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="project_name_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Project Name: PROJECT_NAME</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="output_location_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Output: OUTPUT_LOCATION</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="project_location_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Project Location: PROJECT_LOCATION</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="images_location_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Images: IMAGE_LOCATION</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="number_images_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Number of images:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="sceneLayout"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Press 'A' key to show/hide the axis.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="progressLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="projectToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionOpen"/>
  </widget>
  <widget class="QToolBar" name="featuresToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExtract_Features"/>
   <addaction name="actionMatch_Features"/>
   <addaction name="actionStart_Reconstruction"/>
  </widget>
  <widget class="QToolBar" name="visualizationToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVisualizeBinary"/>
   <addaction name="actionLoad_SLAM"/>
   <addaction name="actionRender_SLAM_Cameras"/>
   <addaction name="actionBuild_with_SLAM"/>
   <addaction name="actionNew_Build"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSearch_Image"/>
   <addaction name="actionBuildToBinary"/>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>Create New Project</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionBuildToBinary">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Build into a binary file</string>
   </property>
  </action>
  <action name="actionExtract_Features">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Extract Features</string>
   </property>
   <property name="toolTip">
    <string>Extract Features</string>
   </property>
  </action>
  <action name="actionMatch_Features">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Match Features</string>
   </property>
   <property name="toolTip">
    <string>Match Features</string>
   </property>
  </action>
  <action name="actionStart_Reconstruction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start Reconstruction</string>
   </property>
   <property name="toolTip">
    <string>Start Recostruction</string>
   </property>
  </action>
  <action name="actionSearch_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search Image</string>
   </property>
   <property name="toolTip">
    <string>Search Image</string>
   </property>
  </action>
  <action name="actionVisualizeBinary">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Visualize binary</string>
   </property>
   <property name="toolTip">
    <string>Visualize a binary (default - out/model-0.binary)</string>
   </property>
  </action>
  <action name="actionRunExampleReconstruction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run Theia's view_reconstruction binary</string>
   </property>
   <property name="toolTip">
    <string>Run Theia's view_reconstruction binary</string>
   </property>
  </action>
  <action name="actionRender_SLAM_Cameras">
   <property name="text">
    <string>Render SLAM Cameras</string>
   </property>
   <property name="toolTip">
    <string>This only renders cameras from co-graph/KeyTrajectory file</string>
   </property>
  </action>
  <action name="actionLoad_SLAM">
   <property name="text">
    <string>Load SLAM data</string>
   </property>
   <property name="toolTip">
    <string>Load SLAM data into Memory and that is it.</string>
   </property>
  </action>
  <action name="actionBuild_with_SLAM">
   <property name="text">
    <string>Fast Build</string>
   </property>
   <property name="toolTip">
    <string>Assuming SLAM data is already loaded, building the project.</string>
   </property>
  </action>
  <action name="actionNew_Build">
   <property name="text">
    <string>New Build</string>
   </property>
   <property name="toolTip">
    <string>using SLAM_Build() method</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
